EnglishFilter/sections/general_extensions.txt

```diff
@@ -234,6 +234,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/77395
+oncehelp.com#%#//scriptlet("prevent-window-open")
```

Comment:
@wout-codes
It looks like `#$?#form[action$="/links/popad"]` is enough, why we need this rule?

Reply:
It doesn't work alone properly. I have tested it multiple times.

Reply:
ok